docs/resources/aviatrix_transit_gateway.md

```diff
@@ -454,12 +452,10 @@ The following arguments are supported:
 * `ha_availability_domain` - (Optional) HA gateway availability domain. Required and valid only for OCI. Available as of provider version R2.19.3.
 * `ha_fault_domain` - (Optional) HA gateway fault domain. Required and valid only for OCI. Available as of provider version R2.19.3.
 * `ha_device_id` - (Optional) Device ID for HA AEP EAT gateway. Required only for AEP HA gateway.
-* `peer_backup_port_type` - (Optional) Peer backup port type for edge transit gateway. Required and valid only for edge gateways AEP and Equinix.
-* `peer_backup_port_index` - (Optional) Peer backup port index for edge transit gateway. Required and valid only for edge gateways AEP and Equinix.
+* `peer_backup_logical_ifname` - (Optional) Peer backup logical interface name for the edge transit gateway (e.g., 'wan0', 'wan1').
 * `peer_connection_type` - (Optional) Connection type for edge transit gateway e.g., "private", "public". Required and valid only for edge gateways AEP and Equinix.
 * `ha_interfaces` - (Optional) A list of WAN/Management interfaces, each represented as a map. Required and valid only for edge transit gateways AEP and Equinix. Each interface has the following attributes:
-  * `type` - (Required) Interface type. Valid values are 'WAN' or 'MANAGEMENT'.
-  * `index` - (Required) Interface index. Valid values are 0,1,2 etc.
+  * `logical_ifname` - (Required) Logical interface name e.g., wan0, mgmt0.
```

Comment:
likewise, this attribute change for transit_gateway should be documented in the feature-changelist-v3.md